ui/package.json

```diff
@@ -11,20 +11,26 @@
   "scripts": {
     "lint": "lerna run --parallel --no-bail lint",
     "clean": "lerna run clean",
-    "prebuild": "yarn lint",
     "build": "lerna run build",
     "start": "lerna run start --scope=@debezium/ui",
     "start:prod": "lerna run start:prod --scope=@debezium/ui",
     "cypress:open": "cypress open",
     "cypress": "cypress run --headless --browser chrome",
+    "format": "lerna run format",
+    "lint-staged": "lerna run lint-staged",
     "unit:test": "lerna run test",
     "i18n": "lerna run i18n --scope=@debezium/ui"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn build"
+      "pre-commit": "yarn run lint-staged"
```

Comment:
yarn build already have the prebuild script to check for lint. you can instead update the rebuild script with this

Reply:
I think it's bit confusing. Here https://github.com/debezium/debezium-ui/blob/main/ui/packages/ui/package.json#L12 `prebuild` runs `yarn clean`  and here https://github.com/debezium/debezium-ui/blob/main/ui/package.json#L14  `yarn lint` also It would be enough to just check for formatting and linting issues instead of building the UI. `cos-ui` follows the same https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/package.json#L144-L151

Reply:
> I think it's bit confusing. Here https://github.com/debezium/debezium-ui/blob/main/ui/packages/ui/package.json#L12 `prebuild` runs `yarn clean` 

this is for the React UI project that we have inside ui/packages 
> and here https://github.com/debezium/debezium-ui/blob/main/ui/package.json#L14 `yarn lint` 

this is the Lerna cmd which runs `yarn lint` inside all the three sub-projects

Reply:
Yes, that's the confusion, we should have identical for both level.

Reply:
actually having identical cmd at both levels for this will result in running the same cmd twice in this project. By having different rebuild scripts we are able to run two diff operations sequentially.

Reply:
I know that but my concern is that why we call `prebuild` at one level to lint the UI and at the second level to clean the UI? We already have https://github.com/debezium/debezium-ui/blob/main/ui/packages/ui/package.json#L20